docs/schema/index-management/index-removal.md

```diff
@@ -162,25 +164,100 @@ g.V().has('name', 'jupiter')
 
 // Disable the "name" composite index
 m = graph.openManagement()
-nameIndex = m.getGraphIndex('name')
-m.updateIndex(nameIndex, SchemaAction.DISABLE_INDEX).get()
+m.updateIndex(m.getGraphIndex('name'), SchemaAction.DISABLE_INDEX).get()
 m.commit()
 graph.tx().commit()
 
-// Block until the SchemaStatus transitions from INSTALLED to REGISTERED
+// Block until the SchemaStatus transitions from ENABLED to DISABLED
 ManagementSystem.awaitGraphIndexStatus(graph, 'name').status(SchemaStatus.DISABLED).call()
 
 // Delete the index using JanusGraphManagement
 m = graph.openManagement()
-nameIndex = m.getGraphIndex('name')
-future = m.updateIndex(nameIndex, SchemaAction.REMOVE_INDEX)
+future = m.updateIndex(m.getGraphIndex('name'), SchemaAction.DISCARD_INDEX)
 m.commit()
 graph.tx().commit()
-
 future.get()
 
+// Block until the SchemaStatus transitions from DISABLED to DISCARDED
+ManagementSystem.awaitGraphIndexStatus(graph, 'name').status(SchemaStatus.DISCARDED).call()
+
+// Index still shows up in management interface as DISCARDED -- it can now be dropped entirely
 m = graph.openManagement()
-nameIndex = m.getGraphIndex('name')
+m.updateIndex(m.getGraphIndex('name'), SchemaAction.DROP_INDEX).get()
+m.commit()
 
+// JanusGraph should issue a warning about this query requiring a full scan
 g.V().has('name', 'jupiter')
-```
\ No newline at end of file
+```
+
+## Executing a manual index removal for mixed indices
+
+!!! info
+    This method applies to the following types of indices:
+
+    - Mixed indices which can not be removed by Janusgraph automatically
+
+If an index backend does not support automatic removal, you will receive an exception when trying to execute `DISCARD_INDEX`.
```

Comment:
IIUC, currently ES supports automatic removal, no? In that case can we document it here?